drivers/flash/spi_nor.c

```diff
@@ -679,6 +679,18 @@ static int spi_nor_read(const struct device *dev, off_t addr, void *dest,
 	return ret;
 }
 
+#if defined(CONFIG_FLASH_RESET)
+static void spi_nor_send_flash_reset(const struct device *dev)
+{
+	acquire_device(dev);
+
+	spi_nor_cmd_write(dev, SPI_NOR_CMD_RESET_EN);
```

Comment:
These functions can fail.  Your new API should provide a return code to the caller.

Reply:
Done